prod-devel-rcar-virtio.yaml

```diff
@@ -711,9 +711,9 @@ parameters:
             builder:
               layers:
                 - "../meta-aos"
-                - "../meta-xt-prod-devel-rcar/meta-aos-tune"
               conf:
-                - [AOS_VIS_PLUGINS, "telemetryemulatoradapter"]
+                - [AOS_VIS_PLUGINS, "storageadapter"]
```

Comment:
This changes should be justified in the commit message like they were described in 559c0d6c2f302050362f58948e674ee800db5d20